nmdc_server/ingest/common.py

```diff
@@ -47,3 +50,14 @@ def extract_extras(
         if key not in fields and key not in exclude:
             values["annotations"][key] = extract_value(value)
     return values
+
+
+def merge_or_delete(ingest_db: Session, query):
+    logger = logging.getLogger()
```

Comment:
Rather than use the root logger, let's move `logger` up to module scope and do `logger = logging.getLogger(__name__)`. (That's the typical practice in Python.)

Reply:
`logging.getLogger()` is used in 7 places. This change seems out of scope for this bugfix, I'll open a different PR.